docs/sources/menu.yaml

```diff
@@ -425,6 +425,15 @@
       link: /enterprise/white-labeling/
```

Comment:
Please delete this file. I deleted it months ago, so I'm not sure why it is back. It has no purpose and is confusing.